daprdocs/content/en/developing-applications/building-blocks/actors/actor-reminders-partitions.md

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
Do we need a new page for this? Or rather just add to this section https://docs.dapr.io/developing-applications/building-blocks/actors/howto-actors/#actor-timers-and-reminders ?

If reminders is becoming a big enough topic we might want to just pull all related content to one docs

Reply:
In my opinion, this page is too detailed to be a section in the other page mentioned in your comment.

Reply:
After discussing with @AaronCrawfis and @msfussell, we decided that after v1.3 we need to refactor and reorganize the Actore section in the docs.  Instead of adding this new page just for partitioning now, please add this info in the existing page. We will be breaking Actors to separate pages soon

Reply:
Are you sure we want this in this page? It is a deep dive into an operational concern. See how the new doc even goes into details into how the data is saved in state store.

Reply:
We want to avoid creating a page that will be removed in a later refactoring. we most likely will not have a page just for partitioning of reminders. An alternative to putting it all in the main overview is to have the new page be more general - about reminders in general (with partitioning being a section inside it)

Reply:
Like I said, my concern is that topics that are deep dives should not in the main overview page. I will merge it for complying with the decision from docs maintainers although this hurts readability for folks that want to get an overview and and suddenly presented with deep technical details that only affects a small subset of actor users.

I just want to leave this comment here for future reference that I am doing it for compliance and not because I think this is right.